docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-monitoring-stack.adoc

```diff
@@ -30,13 +28,12 @@ controller-2.redhat.local   192.168.24.10  mon _admin mgr
 6 hosts in cluster
 ----
 +
-Confirm that the cluster is healthy and both `ceph orch ls` and
+Confirm that the cluster is healthy and that both `ceph orch ls` and
 `ceph orch ps` return the expected number of deployed daemons.
 
-. Review and update the container image registry. If the {Ceph} externalization procedure is executed after the {rhos_prev_long} control
-plane has been migrated, it’s important to consider updating the container
-images referenced in the {CephCluster} cluster config. The current container images point to the undercloud registry, and it might be no longer available. As the undercloud won’t be available in the future, replace the undercloud provided
-images with an alternative registry.
+. Review and update the container image registry:
+[NOTE]
+If you run the {Ceph} externalization procedure after you migrate the {rhos_prev_long} control plane, it is important to consider updating the container images in the {CephCluster} cluster configuration. The current container images point to the undercloud registry, which might not be available anymore. Because the undercloud is not available after adoption is complete, replace the undercloud-provided images with an alternative registry.
```

Comment:
What does the Ceph externalization procedure refer to? Migrating monitoring stack components?

Reply:
@fmount Sorry, the question in number 3 had to do with the note at the end of the module:
"In addition to updating the container images related to the monitoring stack, you must update the configuration entry related to the `container_image_base`. This has an impact on all the {Ceph} daemons that rely on the undercloud images. New daemons are deployed by using the new/default {Ceph} image."

Hopefully it still makes sense to change the last sentence to: "New daemons are deployed by using the new image registry location that is configured in the Ceph cluster."

Reply:
+1 it makes sense, thank you! I'm ok with the rewriting, and we can plan a procedure that can be added in the prerequisites section.